pkgs/development/python-modules/botan3/default.nix

```
@@ -0,0 +1,62 @@
+{
+  lib,
+  stdenv,
+  buildPythonPackage,
+
+  # dependencies
+  botan3,
+
+  # build dependencies
+  setuptools,
+  setuptools-scm,
+  findutils,
+}:
+
+buildPythonPackage rec {
+  pname = "botan3";
+
+  inherit (botan3) src version;
+
+  format = "pyproject";
+
+  build-system = [
+    setuptools
+    setuptools-scm
+  ];
+
+  sourceRoot = "Botan-${version}/src/python";
+
+  postPatch = ''
+    # remove again, when https://github.com/randombit/botan/pull/5040 got
+    # merged
+    cp ${./pyproject.toml} pyproject.toml
+  ''
+  + (
+    if stdenv.hostPlatform.isDarwin then
+      ''
+        botanLibPath=$(${findutils}/bin/find ${lib.getLib botan3}/lib -name 'libbotan-3.dylib' -print -quit)
```

A bit overengineered
```suggestion
        botanLibPath=$(find ${lib.getLib botan3}/lib -name 'libbotan-3.dylib' -print -quit)
```
Findutils are already a part of Stdenv

---

Fixed, thanks!